causalml/metrics/multiqini.py

```diff
@@ -0,0 +1,17 @@
+from maq import MAQ, get_ipw_scores
+
+
+def get_mq_ipw_scores(Y, W, W_hat=None):
+    """Construct evaluation scores via inverse-propensity weighting.
```

Comment:
Is possible to programmatically have this function inherit the docstring from get_ipw_scores? (and similarly below, does MultiQini get MAQ's docstring?)

Reply:
hi @erikcs please see commit below that should add that 👍